software/riscv-tools/Makefile

```diff
@@ -27,8 +27,8 @@
 #
 .DEFAULT_GOAL := help
 
-TARGET_ARCH       := rv32imaf
-TARGET_ABI        := ilp32f
+TARGET_ARCH       := rv64imaf
```

Comment:
need to build with multilib I assume? I don't believe you can compile rv32 with an rv64 toolchain otherwise